truffle/src/com.oracle.truffle.runtime/src/com/oracle/truffle/runtime/OptimizedRuntimeOptions.java

```diff
@@ -158,6 +158,14 @@ public ExceptionAction apply(String s) {
     // TODO: GR-29949
     public static final OptionKey<Long> CompilerIdleDelay = new OptionKey<>(10000L);
 
+    @Option(help = "Before the Truffle runtime submits an OptimizedCallTarget for compilation, it checks for the " +
+                   "compilation activity mode in the host VM. If the activity mode is 'STOP_COMPILATION' because " +
+                   "of a full code cache, no new compilation requests are submitted and the compilation queue is flushed. " +
+                   "After 'StoppedCompilationRetryDelay' milliseconds new compilations will be submitted again " +
+                   "(which might trigger a sweep of the code cache and a reset of the compilation activity mode in the host JVM).",
+                   usageSyntax = "<ms>", category = OptionCategory.EXPERT)
```

Comment:
I think you should still point out in this text that this (currently) only has an effect when running on HotSpot.

Reply:
Hm, not sure if you saw the sentence I've added: "*The option is only supported on the HotSpot Truffle runtime. On runtimes which don't support it the option has no effect*".

Or am I missing something?

Reply:
Sorry, I was confused by the GitHub view which was pinned to a specific commit for some reason.